core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -108,47 +100,18 @@ public MasterWorkerInfo(long id, WorkerNetAddress address) {
    * @param blocks set of block ids on this worker
    * @return A Set of blocks removed (or lost) from this worker
    */
+  // TODO(jiacheng): Do I need to write GuardedBy for member methods like this?
   public Set<Long> register(final StorageTierAssoc globalStorageTierAssoc,
       final List<String> storageTierAliases, final Map<String, Long> totalBytesOnTiers,
       final Map<String, Long> usedBytesOnTiers, final Set<Long> blocks) {
-    // If the storage aliases do not have strictly increasing ordinal value based on the total
-    // ordering, throw an error
-    for (int i = 0; i < storageTierAliases.size() - 1; i++) {
-      if (globalStorageTierAssoc.getOrdinal(storageTierAliases.get(i)) >= globalStorageTierAssoc
-          .getOrdinal(storageTierAliases.get(i + 1))) {
-        throw new IllegalArgumentException(
-            "Worker cannot place storage tier " + storageTierAliases.get(i) + " above "
-                + storageTierAliases.get(i + 1) + " in the hierarchy");
-      }
-    }
-    mStorageTierAssoc = new WorkerStorageTierAssoc(storageTierAliases);
-    // validate the number of tiers
-    if (mStorageTierAssoc.size() != totalBytesOnTiers.size()
-        || mStorageTierAssoc.size() != usedBytesOnTiers.size()) {
-      throw new IllegalArgumentException(
-          "totalBytesOnTiers and usedBytesOnTiers should have the same number of tiers as "
-              + "storageTierAliases, but storageTierAliases has " + mStorageTierAssoc.size()
-              + " tiers, while totalBytesOnTiers has " + totalBytesOnTiers.size()
-              + " tiers and usedBytesOnTiers has " + usedBytesOnTiers.size() + " tiers");
-    }
-
-    // defensive copy
-    mTotalBytesOnTiers = new HashMap<>(totalBytesOnTiers);
-    mUsedBytesOnTiers = new HashMap<>(usedBytesOnTiers);
-    mCapacityBytes = 0;
-    for (long bytes : mTotalBytesOnTiers.values()) {
-      mCapacityBytes += bytes;
-    }
-    mUsedBytes = 0;
-    for (long bytes : mUsedBytesOnTiers.values()) {
-      mUsedBytes += bytes;
-    }
+    mUsage.updateUsage(globalStorageTierAssoc, storageTierAliases,
+            totalBytesOnTiers, usedBytesOnTiers);
 
     Set<Long> removedBlocks;
-    if (mIsRegistered) {
+    if (mMeta.mIsRegistered) {
```

Comment:
I'm not using getters and setters in the `MasterWorkerInfo` to access internal object fields like this. Is it okay?